pkg/prom/registry_test.go

```diff
@@ -132,11 +146,23 @@ func TestRegistryEndpoints(t *testing.T) {
 	})
 
 	t.Run("metrics?target=CCC", func(t *testing.T) {
-		body := get("/metrics?target=CCC")
+		body := get(testRequest{path: "/metrics?target=CCC"})
 		want, dont := []string{}, []string{
 			`fastly_rt_requests_total{datacenter="NYC",service_id="AAA",service_name="Service One"} 1`,
 			`fastly_rt_requests_total{datacenter="NYC",service_id="BBB",service_name="Service Two"} 2`,
 		}
 		checkMetrics(body, want, dont)
 	})
+
+	t.Run("index; accept:text/html", func(t *testing.T) {
+		body := get(testRequest{path: "/", accept: "text/html"})
+		expect(strings.Contains(body, "AAA"), "AAA missing")
+		expect(strings.Contains(body, "BBB"), "BBB missing")
+	})
+
+	t.Run("index; accept:application/json", func(t *testing.T) {
+		body := get(testRequest{path: "/", accept: "application/json"})
+		expect(strings.Contains(body, "AAA"), "AAA missing")
+		expect(strings.Contains(body, "BBB"), "BBB missing")
+	})
```

Comment:
Would it be worth asserting that the `application/json` case returns a response in JSON format?

Reply:
@hermanschaaf good idea 👍 I pushed an update, please take another look